.storybook/manager-head.html

```diff
@@ -1,44 +1,44 @@
 <!-- reference for customizing https://github.com/storybookjs/storybook/issues/11222#issuecomment-646411961 -->
-<link rel="stylesheet" href="https://nav-prod.home.static.arcpublishing.com/css/arcnav.css">
+<link rel="stylesheet" href="https://nav-prod.home.static.arcpublishing.com/css/arcnav.css" />
```

Comment:
`<link>` being a [void element](https://www.w3.org/TR/2011/WD-html-markup-20110113/syntax.html#void-elements) (or an [Empty Element](https://developer.mozilla.org/en-US/docs/Glossary/Empty_element)) doesn't really need the closing slash since it can never have children.  `<link></link>` is invalid, therefore `<link />` is also technically invalid, but the spec ignores self closing tags on void elements.  So it's a toss up whether we keep this.

JSX requires closing, so things like `img` would need it in React.  Closing marks were required under XHTML 1.0, but HTML5 does not.  I don't think we need to revert this, but I thought I'd share.